SkietbaanBE/SkietbaanBE/Helper/SendMail.cs

```diff
@@ -37,6 +37,25 @@ public bool SendEmail(string To, string Subject, Attachment file)
             return true;
         }
 
+        public bool SendPasswordEmail(string To, string Subject, string token)
+        {
+            try
+            {
+                using (MailMessage mail = new MailMessage())
+                {
+                    string url = "http://localhost:3000/resetPassword/";
```

Comment:
Should this URL be hard coded? Do you use environmental variables?

Reply:
currently hard codding it do we need environmental variables??

Reply:
Not sure how you guys do it currently, but this is very useful for when deploying to different environments where it gets the environment url